.tekton/pull-request.yaml

```diff
@@ -124,6 +124,25 @@ spec:
               image: registry.access.redhat.com/ubi8/ubi:latest
               workingDir: $(workspaces.workspace.path)
               script: yum install -y make && make kustomize-build
+      - name: exporters-build-and-push
+        workspaces:
+          - name: workspace
+            workspace: workspace
+        runAfter:
+          - clone-repository
+        taskSpec:
+          workspaces:
+            - name: workspace
+          steps:
+            - name: podman-build
+              image: quay.io/redhat-appstudio/o11y-prometheus-exporters:latest
+              workingDir: $(workspaces.workspace.path)
+              script: |
+                podman build --no-cache -t quay.io/redhat-appstudio/o11y-prometheus-exporters:latest .
+            - name: push
+              image: quay.io/redhat-appstudio/o11y-prometheus-exporters:latest
```

Comment:
Using `latest` is fine for the MVP but it'd also be a good idea to tag images with the commit sha so we can target specific versions for deployment.